templates/test/tr_job_result_failedmodules.html.ep

```diff
@@ -21,8 +21,7 @@
     % }
 
     <span <%= Mojo::ByteStream->new($tooltip) %> class="failedmodule" >
-        <a href="<%= url_for('step', 'testid' => $jobid,
-                             'moduleid' => $failedmodule, 'stepid' => $step) %>">
+        <a href="<%= url_for('test', 'testid' => $jobid) . "?$failedmodule/steps/$step" %>">
```

Comment:
as mentioned in IRC I would like us to have the step controller redirect instead - for backward compat